tests/callbacks/test_quantization.py

```diff
@@ -34,6 +35,7 @@
 def test_quantization(tmpdir, observe: str, fuse: bool, convert: bool):
     """Parity test for quant model."""
     seed_everything(42)
+    assert _TORCH_GREATER_EQUAL_1_10
```

Comment:
Is this here for debugging? This test is not set to run only on 1.10

Reply:
Marking this as a draft then.

> do you have any idea why?

Yes. It's a problem with our `_compare_version` implementation and the torch version:

```python
import operator
from packaging.version import Version

# current behaviour in master
x = operator.ge(Version("1.10.0.dev20210922"), Version("1.10.0"))
print(x)  # False

print(Version("1.10.0.dev20210922").base_version)
# 1.10.0

# potential fix
x = operator.ge(Version(Version("1.10.0.dev20210922").base_version), Version("1.10"))
print(x)
# True
```

We should use `base_version` instead